src/exporter.h

```diff
@@ -0,0 +1,191 @@
+#ifndef EXPORTER_H_
+#define EXPORTER_H_
+
+#include <array>
+#include <cmath>
+#include <fstream>
+#include <iostream>
+#include <sstream>
+#include <string>
+
+#include "backend.h"
+#include "daosoa.h"
+#include "default_force.h"
+#include "inline_vector.h"
+#include "math_util.h"
+#include "param.h"
+
+
+using std::fstream;
+using namespace std;
+
+namespace bdm {
+
+class Exporter {
+ public:
+  template <typename daosoa>
+  void ToFile(daosoa* cells, string filename) const {
+    // This function exports the cell positions into a file,
```

Comment:
Class and function comments must be written before declaration. Otherwise doxygen doesn't recognise it
it also requires `///` comments. Please apply this also for all other occurences